### PR TITLE
fix module path

### PR DESCRIPTION
**Pull request by nevsayno**
change the kernel module path of realtek

### CODE REVIEW

File: functions.sh [hunk 1]
```diff
@@ -387,7 +387,7 @@ backupCurrent ()
             mkdir -p "$rtlwifi_backup_dir"
             tar -czf "$rtlwifi_backup_outfile" -C "$rtlwifi_orig" . > /dev/null 2>&1
         else
-            echo "Could not backup rtlwifi because it could not be found!  Expected at $rtlwifi_orig" >&2
+            echo "Could not backup rtlwifi because it could not be found!  Expected at $rtlwifi_orig"
```
Comment (FreedomBen) [inline functions.sh]: Why remove the redirection to stderr?  Did this cause a problem or was it an accidental commit?

**Comment by FreedomBen** [comment]
@nevsayno Just making sure, did you test this?  I don't have any Ubuntu machines atm so can't test myself.

File: functions.sh (hunk 1 above)
Comment (nevsayno) [reply functions.sh]: sorry, accidental commit.

**Comment by nevsayno** [comment]
I test it and it works for me now. 
```
hgx@nevsayno:~/git/github.com/rtl8188ce-linux-driver$ ./am_i_using_this_driver.sh 
[*] You are running the new rtlwifi
[*] You are running the new rtl8192c_common 
[*] You are running the new rtl8192ce
```

**Pull request closed without merging**